compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeBridge.desktop.kt

```diff
@@ -301,9 +302,15 @@ internal abstract class ComposeBridge {
     }
 
     protected fun updateSceneSize() {
+        val scale = scene.density.density
```

Comment:
```suggestion
        val scale = component.density.density
```

There was a non-valid code here (that probably doesn't cause any bug yet). Because we use AWT property (component.width), we also should use AWT component's density, not scene's one (that can be different, if we expose configuration in the future).

Reply:
As you said, they are equal - it's called only after assignment component's density to scene. But OK, let's replace to `component` here